lib/events/s3sessions/s3handler.go

```diff
@@ -166,13 +166,22 @@ func (l *Handler) Close() error {
 // Upload uploads object to S3 bucket, reads the contents of the object from reader
 // and returns the target S3 bucket path in case of successful upload.
 func (l *Handler) Upload(ctx context.Context, sessionID session.ID, reader io.Reader) (string, error) {
+	var err error
 	path := l.path(sessionID)
-	_, err := l.uploader.UploadWithContext(ctx, &s3manager.UploadInput{
-		Bucket:               aws.String(l.Bucket),
-		Key:                  aws.String(path),
-		Body:                 reader,
-		ServerSideEncryption: aws.String(s3.ServerSideEncryptionAwsKms),
-	})
+	if l.Config.DisableServerSideEncryption {
```

Comment:
Can you change this logic to:

```go
uploadInput := &s3manager.UploadInput{
    Bucket:       aws.String(l.Bucket),
    Key:          aws.String(path),
    Body:         reader,
})
if l.Config.DisableServerSideEncryption == false {
    uploadInput.ServerSideEncryption = aws.String(s3.ServerSideEncryptionAwsKms)
}
_, err := l.uploader.UploadWithContext(ctx, uploadInput)
```

Reply:
Done :)

Reply:
You need either a `!` or `== false` in the `if`statement.